### ISSUE TITLE
WebApp  converts to a TinyML model

### ISSUE THREAD

**Issue by doyu** [Origami-TinyML/tflm_hello_world#38]

**Comment by ArttuLe** [comment]
Is the goal for this task to convert a normal Tensorflow model into TFLite and/or a C source for TFLite micro? Didn't really understand what is meant by "compiling" in this context :)

**Comment by doyu** [comment]
I think that this is same as https://github.com/Origami-TinyML/tflm_hello_world/issues/54
Just generating a flatbuf model binary.
The rest of process should be done by https://github.com/Origami-TinyML/tflm_hello_world/issues/55 since the application logic and a model should be independent.
You should work on human detection of this now without hello now. Did I answer your question here?

**Comment by ArttuLe** [comment]
Yes I am working on the human detection now. And thanks for the clarification :)

**Comment by ArttuLe** [comment]
How should we go about implementing this with the Webapp, since the conversion to .cc model is done via command line ? https://www.tensorflow.org/lite/microcontrollers/build_convert#convert_to_a_c_array 
- Using a python "subprocess" is an option, but currently can't get it to generate the .cc although the subprocess returns True.
- Separate Docker container?
- other?

**Comment by doyu** [comment]
Instead of dumping with xxd, can't just c-program read a binary file of flatbuff tflite directly?

**Comment by ArttuLe** [comment]
Just got the xxd to work :) Should we still keep the xxd or skip it and use the .tflite files directly.

**Comment by doyu** [comment]
xxd is OK for now ;)

**Comment by ArttuLe** [comment]
Should I make a new PR for this or push it to #67 , since this is just one additional method to the existing model training code?

**Comment by doyu** [comment]
Let's leave them separated.
